examples/blockchain/src/main.rs

```diff
@@ -50,21 +58,23 @@ struct Args {
 
 #[tokio::main]
 async fn main() {
+    let time_format =
+        format_description!("[year]-[month]-[day] [hour]:[minute]:[second].[subsecond digits:3]");
     env_logger::builder()
-        .format(|buf, record| {
+        .format(move |buf, record| {
             writeln!(
                 buf,
                 "{} {}: {}",
                 record.level(),
-                Local::now().format("%Y-%m-%d %H:%M:%S%.3f"),
+                OffsetDateTime::now_utc().format(&time_format).unwrap(),
```

Comment:
there is a method `now_local`, doesn't it works here?

Reply:
I must have missed that (it required another feature flag)

I added it, but it doesn't work for me (throws `IndeterminateOffset` error), so I kept UTC as fallback